frontend/routes/Register.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <FormLayout>
+        <Card>
+            <form>
+                <h1>Create an account</h1>
+                <TextInput id="fname" placeholder="First name" required />
+                <TextInput id="lname" placeholder="Last name" required />
+                <TextInput id="email" placeholder="Email" type="email" required />
+                <TextInput id="password" placeholder="Password" type="password" required />
+                <TextInput id="confirm" placeholder="Confirm password" type="password" required />
```

Comment:
Can you add the validation logic for checking that these are the same? I'm not sure the best way to do that in Vue with forms, so it'd be great if you'd like to do research on the best option for that.

Reply:
I looked into some packages for this and vee-validate looks promising.
I *think* it suggests that we have the backend be in charge of input validations like that.

https://vee-validate.logaretm.com/v3/advanced/server-side-validation.html

Reply:
This looks like a good long-term solution. Maybe hold off on validation logic, then, and we can look into adding this validation library in a separate PR?